docs/tutorial.rst

```diff
@@ -1025,16 +1018,86 @@ We'll count up the number of neutral and deleterious mutations in the three regi
             f"and {d} deleterious mutations."
         )
 
-    # From 0 to 1000: 3 neutral mutations and 0 deleterious mutations.
-    # From 1000 to 2000: 1 neutral mutations and 2 deleterious mutations.
-    # From 2000 to 3000: 1 neutral mutations and 0 deleterious mutations.
+    # From 0 to 1000: 50 neutral mutations and 0 deleterious mutations.
+    # From 1000 to 2000: 12 neutral mutations and 20 deleterious mutations.
+    # From 2000 to 3000: 45 neutral mutations and 0 deleterious mutations.
 
 This verifies that there are only deleterious mutations in the center bit.
 
 
+.. _sec_tute_selection_annotation:
+
+3. Simulating selection on exons
+--------------------------------
+
+The catalog also has a certain number of *annotations* available,
+obtained from Ensembl.
+For instance, for humans we have:
+
+.. code-block:: python
+
+    for a in species.annotations:
+        print(f"{a.id}: {a.description}")
+
+    # ensembl_havana_104_exons: Ensembl Havana exon annotations on GRCh38
+    # ensembl_havana_104_CDS: Ensembl Havana CDS annotations on GRCh38
+
+To simulate with the HomSap/Gamma_K17 DFE, applied this time
+to *all* exons on chromosome 21
+and remainder of the chromosome with have only neutral mutations),
+we extract the intervals from the :class:`.Annotation` object
+and use this in :meth:`.Contig.add_DFE`:
+
+.. code-block:: python
+
+    species = stdpopsim.get_species("HomSap")
+    dfe = species.get_dfe("Gamma_K17")
+    contig = species.get_contig("chr20")
+    model = species.get_demographic_model("OutOfAfrica_3G09")
+    samples = model.get_samples(100, 100, 100)  # YRI, CEU, CHB
 
+    exons = species.get_annotations("ensembl_havana_104_exons")
+    exon_intervals = exons.get_chromosome_annotations("chr20").astype("int")
```

Comment:
i wonder if we should add a helper function here to apply the exonic intervals from a chrom / annotation. could save a line of code?

Reply:
https://github.com/popsim-consortium/stdpopsim/issues/1079#issuecomment-957110511

Reply:
ah nice